viewer/source/assets/shaders/outlines.vert.glsl

```diff
@@ -0,0 +1,51 @@
+#version 450
+
+layout(location = 0) in vec4 inPosition;
+layout(location = 1) in vec2 inOffset;
+layout(location = 2) in vec2 inSize;
+layout(location = 3) in vec3 inInfo;
+
+layout(set = 1, binding = 0) uniform Params {
+    float resolution;
+    float maxDepth;
+} params;
+
+layout(location = 0) out vec3 outInfo;
+
+void main() {
+    float lineWidth = 1.0 / params.resolution;
+
+    if (inInfo.y < 3) {
```

Comment:
Maybe you have a better formula for the line width scaling between different layers?